templates/about/getting-there.html

```diff
@@ -20,6 +20,13 @@ <h3>By Car</h3>
 
 <p>Once you're near the site, you'll see yellow temporary road signs for EMF - follow these and ignore your sat nav if it tells you differently.</p>
 
+<h4>Electric Vehicles</h4>
+<p>There will be no provision for EV charging at the event this year due to both a substantial cost in fuel for the generators on-site, as well as a predicted increase in demand.</p>
```

Comment:
```suggestion
<p>Owing to the increased price of providing electricity at EMF, and the increasing number of electric
    vehicles, we do not plan to provide generally-available electric vehicle charging at EMF 2022.
    We ask that you arrive at EMF with enough charge to leave at the end of the event.</p>
```

(Keeps it consistent with wording on the power page)